docs/setup-deploy-nixops.adoc

```diff
@@ -0,0 +1,26 @@
+= Deploy with NixOps
+
+1.
+include::autogenerate/Bootstrap.adoc[]
+
+2.
+include::autogenerate/Target.adoc[]
+
+3.
+Get a cluster join token.
+
+--
+- In the https://hercules-ci.com/dashboard[dashboard], find the account for which you would like to deploy the agent,
+- Click the "Agents" button and the button in "Generate token" tab. This produces a private token that should be protected like a password.
+- Copy the token into a plain text file `cluster-join-token.key` in the same directory. Protect this file like a password.
+--
+
+3.
+Write a link:#binarycachespath[binary caches file], `binary-caches.json` in the same directory as `hercules-ci-agents.nix`.
```

Comment:
It's a lot more complicated than that, maybe it should include to Cachix file?

Reply:
This is the NixOps deployment, which I only moved. Are you sure about this?

Reply:
Ah this one is fine :)